### PR TITLE
UnitTestHomeWork

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: Lesson10/src/test/java/Lesson10/FruitShopUnitTest.java [hunk 1]
```diff
@@ -76,18 +78,71 @@ public void testAllFreshNullTwo(){
...
+        expected.add(strawberry);
+        expected.add(pear);
+        List<Fruit> actual = fruitShop.reSetPricexRealization(30,30,FruitType.APPLE);
+        assertEquals(expected,actual);
+    }
+
+    @Test
+    public void testReSetPriceTrueTest(){
+        List<Fruit>actual  = fruitShop.reSetPricexRealization(30,30,FruitType.APPLE);
+        assertTrue(actual.size() == 0);
```
Comment (mkharkhu) [inline Lesson10/src/test/java/Lesson10/FruitShopUnitTest.java]: looks good

File: Lesson10/src/test/java/Lesson10/FruitShopUnitTest.java [hunk 2]
```diff
@@ -76,18 +78,71 @@ public void testAllFreshNullTwo(){
...
+    }
+
+    @Test
+    public void testAllFruitOfTypeTrueTest(){
+        List<Fruit> actual = fruitShop.allFruitOfFruitType(FruitType.APPLE);
+        assertTrue(actual.size()==0);
+    }
+    @Test(expected = NullPointerException.class)
+    public  void testAllFruitTypeNullTest(){
+        fruitShop.allFruitOfFruitType(FruitType.APPLE);
```
Comment (mkharkhu) [inline Lesson10/src/test/java/Lesson10/FruitShopUnitTest.java]: The two previous tests look very similar. Is there any logic in this?

File: Lesson10/pom.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <maven-assembly-plugins.version>3.1.1</maven-assembly-plugins.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
```
Comment (SergeyPasko) [inline Lesson10/pom.xml]: ????

Comment (Daniluko) [reply Lesson10/pom.xml]: I must delete source and target 8?

**Pull request merged**